pyaedt/q3d.py

```diff
@@ -554,3 +555,116 @@ def __init__(
             close_on_exit,
             student_version,
         )
+
+    def create_rectangle(self, position, dimension_list, name="", matname=""):
+        """
+        Create a rectangle.
+
+        Parameters
+        ----------
+        position : list
+            List of  [x, y] coordinates for the starting point of rectangle
+        dimension_list : list
+            List of [width, height] dimensions.
+        name : str, optional
+            Name of the rectangle. The default is ``None``, in which case
+            the default name is assigned.
+        matname : str, optional
+            Name of the material. The default is ``None``, in which case
+            the default material is assigned.
+        Returns
+        -------
+        pyaedt.modeler.Object3d.Object3d
+            3D object.
+
+        """
+        pos = self.modeler.Position(*position, 0)
+        return self.modeler.primitives.create_rectangle(pos, dimension_list=dimension_list, name=name, matname=matname)
+
+    def assign_single_signal_line(self, target_objects, name="", solve_option="SolveInside", thickness=None, unit="um"):
+        """
+        Assign conductor type to sheets.
+
+        Parameters
+        ----------
+        name : str
+            Name of the conductor.
+        target_objects : list
+            List of Object3D.
+        solve_option : str
```

Comment:
```suggestion
        solve_option : str, optional
```

Reply:
Please reorder the parameters to match the order in which they are referenced in the function expression.